frontend/catalyst/jax_tracer.py

```diff
@@ -1073,8 +1080,11 @@ def is_leaf(obj):
                     output = return_values_flat
                     trees = return_values_tree
 
-                qrp_out = trace_quantum_tape(tape, device, qreg_in, ctx, trace)
-                meas, meas_trees = trace_quantum_measurements(device, qrp_out, output, trees, tape)
+                mcm_config = (
+                    getattr(qnode, "_tmp_mcm_config", None) or qnode.execute_kwargs["mcm_config"]
+                )
```

Comment:
I'm confused, is `_tmp_mcm_config` set anywhere? If so I couldn't find it 😅 Also thought it wouldn't be required anymore since the one shot QNode can be freely mutated without affecting the user qnode.

Reply:
I think I may have forgotten to remove this after I changed the implementation.